projects/rero/ng-core/src/lib/record/editor/horizontal-wrapper/horizontal-wrapper.component.ts

```diff
@@ -22,9 +22,8 @@ import { FieldWrapper } from '@ngx-formly/core';
   template: `
     <div class="form-group m-0">
       <div class="d-flex">
-        <label [attr.for]="id" class="text-nowrap mr-2 col-form-label-1" *ngIf="to.label" [tooltip]="to.description">
-          {{ to.label }}
-          <ng-container *ngIf="to.required && to.hideRequiredMarker !== true">*</ng-container>
+        <label [attr.for]="id" class="text-nowrap1 mr-2 col-form-label-1" *ngIf="to.label" [tooltip]="to.description">
+          {{ to.label }}<ng-container *ngIf="to.required && to.hideRequiredMarker !== true">&nbsp;*</ng-container>
```

Comment:
Cosmeic/For a next PR : I think the "required star" should be highlighted with red and/or bold font.

Reply:
> Cosmetic/For a next PR : I think the "required star" should be highlighted with red and/or bold font.

This should need to be discussed with the PO. Thanks for your fast review.